### PR TITLE
Sales

### PR DESCRIPTION
**Pull request by dustMason**
I know there's still another PR open but just wanted to keep things moving along. Basic implementation of Account.Sales here.

### CODE REVIEW

File: lib/lightspeed/sale_line.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+require_relative 'resource'
+require_relative 'item'
+
+module Lightspeed
+  class SaleLine < Lightspeed::Resource
+    alias_method :archive, :destroy
```
Comment (dnl) [inline lib/lightspeed/sale_line.rb]: As these don't appear to be archiveable, I'd rather there was no archive method on these, and keep the archive method only on those things that can be archived.

File: lib/lightspeed/sale_lines.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require_relative 'collection'
+
+require_relative 'sale_line'
+
+module Lightspeed
+  class SaleLines < Lightspeed::Collection
+    alias_method :archive, :destroy
```
Comment (dnl) [inline lib/lightspeed/sale_lines.rb]: and here.

File: lib/lightspeed/sale_line.rb (hunk 1 above)
Comment (dustMason) [inline lib/lightspeed/sale_line.rb]: yeah absolutely, included that by mistake

File: spec/lightspeed/sale_spec.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe Lightspeed::Sale do
+  setup_client_and_account
+
+  subject do
+    Lightspeed::Sale.new(context: account, attributes: { "saleID" => 2 })
+  end
+
+  # it "can fetch a sale's Customer" do
```
Comment (dnl) [inline spec/lightspeed/sale_spec.rb]: why are these tests commented out?

**Comment by dnl** [comment]
After merging the other and tweaking it slightly I've rendered this conflicting. sorry. 

I'll happily merge otherwise.

File: spec/lightspeed/sale_spec.rb (hunk 3 above)
Comment (dustMason) [inline spec/lightspeed/sale_spec.rb]: because those relationships haven't been defined yet. unfinished.

**Comment by dustMason** [comment]
this means i need to rebase against master and clean this up? i'll see if i can get to this soon

File: spec/lightspeed/sale_spec.rb (hunk 3 above)
Comment (dnl) [inline spec/lightspeed/sale_spec.rb]: cool. can you remove them from this PR, and add them when they are defined?

Comment (dustMason) [inline spec/lightspeed/sale_spec.rb]: sure will do

**Comment by dnl** [comment]
don't worry too much about the cleaning up the commit history.

**Comment by dustMason** [comment]
This should be good to go now!

**Comment by dustMason** [comment]
Coming back to this now that the holidays are over. Anything more to be done here?

**Comment by dnl** [comment]
Thanks for the reminder. This is merged now.

**Comment by dustMason** [comment]
thanks!

**Pull request merged**